src/applications/vaos/new-appointment/components/ClinicChoicePage.jsx

```diff
@@ -45,142 +40,111 @@ const initialSchema = {
     },
   },
 };
-
 const uiSchema = {
   clinicId: {
     'ui:widget': 'radio',
   },
 };
-
 const pageKey = 'clinicChoice';
-
-export class ClinicChoicePage extends React.Component {
-  componentDidMount() {
-    this.props.openClinicPage(pageKey, uiSchema, initialSchema);
+export function ClinicChoicePage({
+  schema,
+  data,
+  facilityDetails,
+  typeOfCare,
+  clinics,
+  eligibility,
+  canMakeRequests,
+  openClinicPage,
+  updateFormData,
+  facilityDetailsStatus,
+  pageChangeInProgress,
+  routeToNextAppointmentPage,
+  routeToPreviousAppointmentPage,
+}) {
+  const history = useHistory();
+  const typeOfCareLabel = formatTypeOfCare(typeOfCare.name);
+  const usingUnsupportedRequestFlow =
+    data.clinicId === 'NONE' && !canMakeRequests;
+  useEffect(() => {
+    document.title = `${getPageTitle(schema, typeOfCare)} | Veterans Affairs`;
+    openClinicPage(pageKey, uiSchema, initialSchema);
     scrollAndFocus();
```

Comment:
It looks like the previous intent of this page was to again scrollAndFocus when we display an h1, which happens after loading has finished.

I think I would create another useEffect with `schema && facilityDetailsStatus !== FETCH_STATUS.loading` as a dependency

Reply:
Done